<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScannerWindow</class>
 <widget class="QDialog" name="ScannerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>170</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Card Scanner</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QComboBox" name="cmbInterfaces">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="btnScan">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Scan</string>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/resources/images/scan.png</normaloff>:/resources/images/scan.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>17</height>
        </size>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="lblInterfaces">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Interfaces</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="3">
      <widget class="QListWidget" name="lstCards"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lblIP">
       <property name="text">
        <string>Connect to</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="txtIP">
       <property name="focusPolicy">
        <enum>Qt::TabFocus</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="3">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="btnConnect">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Connect</string>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/resources/images/connect.png</normaloff>:/resources/images/connect.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>17</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txtIP</tabstop>
  <tabstop>btnConnect</tabstop>
  <tabstop>lstCards</tabstop>
  <tabstop>cmbInterfaces</tabstop>
  <tabstop>btnScan</tabstop>
 </tabstops>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
